test/SushiSwapPool.test.ts

```diff
@@ -217,7 +217,7 @@ describe("SushiSwapPool", () => {
       expect(await newSushi.pair()).to.equal(await sushiUGOVPool.pair());
     });
 
-    it("should add pool and earn sushi", async () => {
+    it.skip("should add pool and earn sushi", async () => {
```

Comment:
I don't understand why we would skip an old test that was passing ...we can remove the `skip`

Reply:
> > **DEPLOYMENT MasterChefV2.1 PROCESS**
> > 1/ Deploy MasterChefV2.1 SmartContract to mainnet with script
> > `npx hardhat deploy --tags MasterChefV2.1 --network mainnet`
> > _To be launch with private key of UBQ manager_
> > 2/ Tests :
> > ```
> > * Check initialization with 5 deposits is ok
> > 
> > * Check simulateMigrate on new deployment (after local fork)
> > 
> > * Other manual checks
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 3/ Set manually UBQ_MINTER_ROLE to new MasterChefV2
> > 4/ Set manually setUGOVPerBlock to one (1e18)
> > 5/ Set manually new MasterChefV2 address in manager
> > _manually : can be done via etherscan or other means_
> 
> I think we should set the UGOVPerBlock in the constructor to save some gas ;)

we're here to chew bubble gum and burn some gas...and we're all out of bubble gum

Reply:
both estimation gas problem after london hardfork + too long calculation on MasterChefV1 (endless on local computer), on this line : 
`await masterChef.connect(sushiChef).add(totAllocPoint, uGOVPair, true);`

that is Masterchef V1 problem...